app/assets/javascripts/modules/remove-filter.js

```diff
@@ -34,8 +34,7 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
       }
       else if (inputType == 'text' || inputType == 'search') {
         $input.val(currentVal.replace(removeFilterValue, '').replace(/\s+/g,' ').trim()).trigger({
-          type: "change",
-          suppressAnalytics: true
```

Comment:
Removing this is causing the analytics event for removing a facet tag to fire twice (which is why `suppressAnalytics` was introduced). We still need to do some sort of check to make sure we don't fire events twice in this scenario.

Reply:
So I was chatting with Haur about it and we initially agreed that it was fine for it to log two events:facetTagRemoved and filterRemoved (i.e one of clicking on the facetTag  and the other for any filter changes). 

However this afternoon we were going through all the various facets and realised there is an inconsistency with this approach between removing text in the keywords, clearing select list or unticking checkboxes. 

So I'll try and introduce this again see if there is a better solution.

Reply:
I've reverted these changes as you have recommended.  What I have done was ensure that we use `suppressAnalytics` more across the components and the events.

I think before it was only being used for text inputs but it should of been used for checkboxes/radios and any other form elements that are changed when a user clicks on "Remove facet label"